docs/conf.py

```diff
@@ -9,7 +9,8 @@
 if IS_READTHEDOCS:
     docs_dir = os.path.dirname(__file__)
     project_dir = os.path.join(docs_dir, '..')
-    subprocess.check_call('npm install', cwd=project_dir, shell=True)
+    subprocess.check_call('npm install --global yarn', cwd=project_dir, shell=True)
```

Comment:
be aware this is very old npm and old node (I don't recall the versions now, but it caught me surprised)